srcpkgs/stig/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'stig'
+pkgname=stig
+version=0.9.0a
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid python3-urwidtrees python3-xdg"
```

Comment:
fold this at 80 columns.

you can use `fold -w 80 -s` to get it folded, add a whitespace on the new line.